oak-core/src/test/java/org/apache/jackrabbit/oak/plugins/index/AsyncIndexUpdateTest.java

```diff
@@ -307,6 +307,142 @@ public void testAsyncPause() throws Exception {
         assertEquals(ImmutableSet.of("testRoot"), find(lookup, "foo", "abc"));
     }
 
+    @Test
+    // this test is very sensitive to timing
+    // - effectively it can only be run in standalone mode
+    @Ignore
```

Comment:
Unfortunate that the test is timing sensitive - but the downside with ignoring it here is that it basically will never be executed ever again ..

Reply:
You are right of course.

What we could do is commit the change, and open a follow-up issue to improve the test case, so it is not dependent on timing. That way, the immediate problem is fixed for now. The risk of that, of course, is that we don't have time to ever work on the follow-up item...

Reply:
+1, follow-up ticket sounds good to me